x-pack/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -5,8 +5,9 @@
  */
 
 import { GraphQLSchema } from 'graphql';
-import { Request, ResponseToolkit, Server } from 'hapi';
+import { Request, ResponseToolkit, Server } from 'src/server/kbn_server';
```

Comment:
__question:__ why can't we use relative paths here and point to the root `index.d.ts` with `Legacy` namespace?

Reply:
See comment above, you can now do `import { Legacy } from 'kibana'`